docs/reduction-in-locks.md

```diff
@@ -0,0 +1,100 @@
+
+* Set a backup lock on the server to prevent any `DDL` operations to ensure no new tables are created, renamed, or dropped during the backup.
+
+* Copy redo logs from the checkpoint up to the current Log Sequence Number (LSN) and start tracking new entries.
+
+* Identify and copy the `.ibd` files.
+
+* Copy non-InnoDB files.
+
+* Gather a sync point from all engines (InnoDB LSN, binlog, Global Transaction Identifier (GTID), etc.) by querying the log status.
+
+* Stop the redo thread once it copies at least up to sync point.
+
+* Release the backup lock on the server.
+
+If the backup lock is disabled with the [`--lock-ddl=OFF`](./xtrabackup-option-reference.md/#lock-ddl) option, a backup continues while concurrent DDL events are executed. These backups may be invalid and may fail at either the `backup` or the `--prepare` step.
+
+## The `--lock-ddl=REDUCED` option overview
+
+[Percona XtraBackup 8.4.0-2](./release-notes/8.4.0-2.md) adds the [`--lock-ddl=REDUCED`](./xtrabackup-option-reference.md/#lock-ddl) option to reduce the time the server remains locked by `xtrabackup`. Now, you can execute more `DDL` operations concurrently while the backup is in progress. The backup lock is now taken after copying the `.ibd` files and before copying the `non-InnoDB` files.
+
+### Benefits
+
+The `--lock-ddl=REDUCED` option key features are as follows:
+
+* Reduced locking: Rather than holding the backup lock throughout the entire backup process, it is only acquired for a very short period.
+
+* DDL statements: The server remains accessible for `DDL` operations during the backup process.
+
+* Consistency: Consistency is maintained by using the redo log to track file operations. During the backup process, new metadata files are created to ensure consistency. In the `--prepare` phase, these metadata files are processed first to bring the database to a consistent state.
+
+The new backup design includes the following phases:
+
+### Phase 1: `Operations performed without the lock`
+
+1. Copy the redo logs from the checkpoint up to the current `LSN` and start tracking new entries.
+2. Start the redo log thread.
+3. Track file operations from the redo log by parsing the `MLOG_FILE_*` records.
+4. Copy the `.ibd` files.
+
+### Phase 2: `Operations performed under the lock`
+
+1. Acquire the backup lock (Lock Tables For Backup (LTFB)/Lock Instance For Backup (LIFB)) to prevent new DDL operations, such as creating or altering tables.
+2. Check the file operations that were tracked and recopy the tablespaces.
+3. Create additional `meta` files to perform the required actions (deletions or renames) on the already copied files. This approach ensures that the backup remains consistent and accurate without disrupting the streaming process. This step is required for taking streaming backups.
+    
+    The meta files are the following:
```

Comment:
This table seems to out of place - could you move it to after the description of the procedure?

Reply:
moved